packages/web-app-files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -46,35 +88,14 @@
       />
     </template>
     <hr />
-    <div v-if="expirationSupported" class="oc-mt-m">
-      <div class="uk-position-relative">
-        <oc-datepicker
-          id="files-collaborators-collaborator-expiration-input"
-          :key="`collaborator-datepicker-${enteredExpirationDate}`"
-          :date="enteredExpirationDate"
-          :max-datetime="maxExpirationDate"
-          :min-datetime="minExpirationDate"
-          :label="datePickerLabel"
-          @input="setExpirationDate"
-        />
-        <div
-          v-if="canResetExpirationDate"
-          id="files-collaborators-collaborator-expiration-delete"
-          v-oc-tooltip="expirationDateRemoveTooltip"
-          class="uk-position-small uk-position-center-right oc-cursor-pointer"
-          uk-close
-          @click="resetExpirationDate"
-        />
-      </div>
-      <hr />
-    </div>
   </div>
 </template>
 
 <script>
 import { mapGetters } from 'vuex'
 import { DateTime } from 'luxon'
 import get from 'lodash-es/get'
+import DatePicker from 'v-calendar/lib/components/date-picker.umd'
```

Comment:
Why are you adding a different DatePicker library than the one we're using in the ODS here directly vs bringing it to the ODS? Maybe there was a team decision but I then might've missed it:)

Reply:
> Move v-calendar to ODS

I have it as a task to move it to ODS. I pushed it here for now only to see how the CI will behave...

Reply:
Thanks for clarification, I'm curious about the new implementation regarding both a11y and "getting rid of UIkit ;)"